composer.json

```diff
@@ -23,6 +23,7 @@
         "ext-xml": "*"
     },
     "require-dev": {
+        "composer/xdebug-handler": "^3.0.5",
         "friendsofphp/php-cs-fixer": "^3.66.0",
```

Comment:
Adding a dependency can be convenient but the source must be trustfully. 

Of course, composer is safe but is necessary?

The more decoupled and independent something is, the better.